### PR TITLE
add views/.login.html file and update index.js

### PR DESCRIPTION
**Pull request by Birek100**
Add file login.html and move there a part of content from index.js file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,38 +1,14 @@
...
-<footer>
-All rights reserved &copy
-</footer>
 
-</div>
-</body>
-</html> `))
+const data = fs.readFileSync('./views/login.html','utf8' ,(err, data) => {
+  if (err) throw err;
+  console.log(data);
```
Comment (oskarszura) [inline index.js]: You don't need to console log `data` here, logging html templates makes no sense.

File: index.js [hunk 2]
```diff
@@ -1,38 +1,14 @@
...
-<footer>
-All rights reserved &copy
-</footer>
 
-</div>
-</body>
-</html> `))
+const data = fs.readFileSync('./views/login.html','utf8' ,(err, data) => {
+  if (err) throw err;
+  console.log();
```
Comment (oskarszura) [inline index.js]: No, I've meant, you don't need to have `console.log` at all, what's the reason standing behind having just an empty `console.log` here? ;)

File: index.js (hunk 1 above)
Comment (Birek100) [reply index.js]: I think it's fixed

File: index.js (hunk 2 above)
Comment (oskarszura) [inline index.js]: No it's not fixed, and actually, the way you're using `readFileSync` is a bit wrong, take a look at the documentation:
`fs.readFileSync(path[, options])`, there's no callback at all. Please adjust it.

File: index.js [hunk 3]
```diff
@@ -1,38 +1,13 @@
...
-</div>
-</section>
-<footer>
-All rights reserved &copy
-</footer>
 
-</div>
-</body>
-</html> `))
+const data = fs.readFileSync('./views/login.html','utf8' ,(err, data) => {
```
Comment (oskarszura) [inline index.js]: Take a look at this part of the code...

```
const data = fs.readFileSync('./views/login.html','utf8' ,(err, data) => {
   if (err) throw err;
`});
```

it's SYNCHRONOUS way of using `readFile`, callback is a function that is called when some ASYNCHRONOUS task is accomplished. You still have this callback here. Please compare your usecase with the documentation and check existing examples in the documentation.

**Comment by Birek100** [comment]
const data = fs.readFileSync('./views/login.html','utf8' ,(err, data) => {});
It should be like this??

**Comment by oskarszura** [comment]
@Birek100 - where in the docs it says to use a function 
```
(err, data) => {})
```
? It doesn't... you have to have a firm understanding of that the docs says. Do not guess! ;) 

Read the docs:
> fs.readFileSync(path[, options])
>
> path <string> | <Buffer> | <URL> | <integer> filename or file descriptor
> options <Object> | <string>

first parameter is a path (string) - it's ok,
second parameter is not an encoding code nor third is a callback, it's an object with some properties.

**Comment by Birek100** [comment]
const data = fs.readFileSync('./views/login.html','utf8');
I think this should be ok

**Comment by oskarszura** [comment]
yes, seems to be ok - please update the PR accordingly.

**Comment by Birek100** [comment]
It's updated

**Comment by oskarszura** [comment]
Ok, 👍 , please merge it! :)

**Pull request merged**